generator/internal/client_generator.cc

```diff
@@ -91,7 +91,7 @@ Status ClientGenerator::GenerateHeader() {
     "$class_comment_block$\n"
     "class $client_class_name$ {\n"
     " public:\n"
-    "  explicit $client_class_name$(std::shared_ptr<$connection_class_name$> connection);\n"
+    "  explicit $client_class_name$(std::shared_ptr<$connection_class_name$> connection, Options options = {});\n"
```

Comment:
I think using an unqualified `Options` here is Okay, because this code lives in `google::cloud::$library$::v1_X_Y` any `Options` class or struct (say one generated by protos) would be in `google::cloud::$library$::v1` or some such namespace.

Is my reasoning correct?

Reply:
A little simpler that that, I think, in that I don't think we're expecting to find anything other than `google::cloud::Options`.  We already do this for `Options` in existing library-qualified namespaces, and also for other "common" symbols.